ITC205_BaseLine_Code/src/Main.java

```diff
@@ -1,5 +1,7 @@
 import java.text.SimpleDateFormat;
 import java.util.Scanner;
+//Ready for static review
```

Comment:
D20  - line 102 / 103 – change “e” → “exception”

Reply:
D20 - Amended 'e' to 'exception' on all lines of relevant code.

Reply:
Confirmed Fixed - Facilitator